eslint-rules/tests/component_deprecation.json

```diff
@@ -7,6 +7,14 @@
       {
         "prop": "url",
         "message": "Please use the 'imageSource' prop instead."
+      },
```

Comment:
btw, component depreciation here is merely for example, no need to update it

Reply:
🤣 revereted